src/main/java/com/kryptnostic/conductor/rpc/ConductorElasticsearchApi.java

```diff
@@ -32,17 +32,32 @@
 import com.google.common.base.Optional;
 
 public interface ConductorElasticsearchApi {
+    
+    // settings consts
+    final String NUM_SHARDS = "number_of_shards";
+    final String NUM_REPLICAS = "number_of_replicas";
+    final String ANALYSIS = "analysis";
+    final String FILTER = "filter";
+    final String ANALYZER = "analyzer";
+    final String ENCODER = "encoder";
+    final String REPLACE = "replace";
+    final String TOKENIZER = "tokenizer";
+    final String STANDARD = "standard";
+    final String LOWERCASE = "lowercase";
+    final String PHONETIC = "phonetic";
+    final String METAPHONE = "metaphone";
```

Comment:
Maybe worthwhile to batch `METAPHONE`, `METAPHONE_FILTER` and `METAPHONE_ANALYZER` into one single enum constant, which will be passed in by the relevant field in property type instantiation.